src/recipes/heartbeat-by-user-first-impression/config.js

```diff
@@ -12,15 +12,71 @@
 
 "use strict";
 
-let million = Math.pow(10,6);
-let thousand = Math.pow(10,3);
-let percent = 0.01;
-let days = 24 * 60 * 60 * 1000;
+const VERSION=33;
 
-/*
-  days since last ask?  -- function or constant?
+/** EngagementUrl Rules
 
-*/
+  Notes:
+  1.  first match will win.  PUT MORE SPECIFIC RULES FIRST.
+      Note: NO automated check of rule order.
+  2.  'qsurvey' urls will postpend all the right query args.  others won't.
+      (Write a bug if this is a problem!)
+  3.  If you need more fields, dig into `src/common/personinfo.js` .
+  4.  `npm run surveys` will attempt to run some common configs through
+      the rules.
+  5.  `test/recipe/test-heartbeat-by-user-first-impression.js` does some
+      additional linting.
+ */
+
+const engagementRules = [
+  // en, aurora.
+  {
+    alias: '^en, ^aurora',
+    rule: {
+      locale: /^en/,
+      channel: /^aurora/
+    },
+    urls: [
```

Comment:
I mentioned before, that I think that this should be objects to make sure url and break is 1:1 (not sure if you missed it or just disagreed)
urls [{url: <>, break: <>}, {url: <>, break: <>}, ... ]

Perhaps breaks should be sample pcts?  Specifying 20, 30, and 50 might be more intuitive than 20, 50, and 100.  Although we would need to test that sum(breaks) <= 100

As it stands do we test that break(n) < break(n-1)?  <ight be a decent TODO.